arch/riscv/CMakeLists.txt

```diff
@@ -2,4 +2,10 @@
 
 add_subdirectory(core)
 
+if(CONFIG_64BIT)
+zephyr_compile_options(-mabi=lp64 -march=rv64ima -mcmodel=medany)
+set_property(GLOBAL PROPERTY PROPERTY_OUTPUT_FORMAT elf64-littleriscv)
+else()
+zephyr_compile_options(-mabi=ilp32)
```

Comment:
This is probably good enough for now, but it exposes the fact that we really need a way to:
- Derive the RISC-V architecture from the DeviceTree
- Choose a suitable ABI to match
- Make sure that the toolchain supports that combination of arch/ABI

Reply:
@nategraff-sifive how would you recommend the toolchain for riscv64 get configured?